docs/application/web/get-started/addon/first-addon.md

```diff
@@ -0,0 +1,530 @@
+
+![JavaScript file for resume](media/js_for_resume.png)
+
+You need to use `addonapi` which includes APIs for addons. To show the pop-up, you can use `addonapi.window.showMessageBox()`. Here's a usage of it.
+
+```
+const addonAPI = require('addonapi');
+
+resume(winId) {
+    const options = {
+        type: 'info',
+        buttons: ['OK'],
+        title: 'Resume',
+        message: 'Hello. Resume occurred!!'
+    };
+    addonAPI.window.showMessageBox(winId, options);
+}
+```
+    
+
+By adding the lines above, the JavaScript file will look like this :
+
+![Code for resume pop-up](media/hello_resume_code.png)
+
+With the changed file, you need to build a signed package again.
+
+Before installing the newly built package, you have to remove the installed addon first.
+
+In Addon Settings App, you can see the 'X' button in red box in the row of the installed addon list. To remove the installed addon, you need to press the button.
+
+Now, the changed addon package with pop-up on resume event will be installed if you follow the steps you did with the template package before.
+
+When you run the Basic UI web application, you will see the pop-up as below.
+
+**Figure: Addon Pop-up on Resume Event in the Basic UI**
+
+![Showing pop-up in Basic UI](media/hello_resume.png)
+
+
+### Loading an HTML File from Addon
+
+In an addon, you can load your own html file if you need. We are going to write an HTML file which shows `Hello, I'm an Addon`.
+
+All files related to an addon should be located under `shared\res` of your addon project. Therefore, the added HTML file should be there, too.
+
+![Add a new file](media/new_file.png)
+
+![Generate the HTML file](media/generate_hello_html.png)
+
+Let's write a HTML as below. That will be named as `hello.html` :
+
+```
+<!DOCTYPE html>
+<html>
+<head>
+    <meta name="viewport" content="user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, width=device-width">
+</head>
+<body>
+    <h1>Hello, I'm an Addon</h1>
+</body>
+</html>
+```
+
+The HTML file should be loaded in the addon. Here, we try to do it when the resume event occurs.
+
+To do that, you will use `addonapi.window.loadURL()`. Then, the `rusume()` will look like this
+
+```
+resume(winId) {
+    console.log('TmplDefault.resume');
+    addonAPI.window.loadURL(winId, 'file://' + __dirname + '/hello.html');
+    const options = {
+        type: 'info',
+        buttons: ['OK'],
+        title: 'TcResume',
+        message: 'Hello. Resume occurred!!'
+    };
+    addonAPI.window.showMessageBox(winId, options);
+}
+```
+
+Again, you need to remove the installed addon, build a package with the HTML file and changed JavaScript file, and install the new package.
+
+If there's no issue, you will see the screen as below.
+
+![Resume with HTML and pop-up](media/resume_html_popup.png)
+
+### Return to the main page of the web application
```

Comment:
**Note**: From here, the content is written more in a blog language. Request the author to please maintain the writing style as per the Tizen docs. Also, since this file is too huge, could we have the Example part in a separate page?
@mijin85cho, please provide your opinion as well. what shall we do here? Shall we re-write everything so that it aligns to Tizen doc style or wait for the author to update and then review?

Reply:
done to change the language. please check it out.